app/lib/settings/src/subpages/my_profile/change_type_of_user/change_type_of_user_repository.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2024 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:cloud_functions/cloud_functions.dart';
+import 'package:user/user.dart';
+
+class ChangeTypeOfUserRepository {
```

Comment:
Its not really a Repository, lets just call it `ChangeTypeOfUserService` instead I'd say.

Reply:
Or anything else, just not Repository since it does not follow the Repository pattern.